src/legacy/core_plugins/metrics/server/lib/vis_data/helpers/bucket_transform.js

```diff
@@ -78,11 +78,12 @@ export const bucketTransform = {
   },
   static: bucket => {
     checkMetric(bucket, ['value']);
+    const isDecimalValue = !Number.isInteger(Number(bucket.value));
```

Comment:
Maybe we can move it into helper 'method' to reuse it in future. 
Also looks like it returns 'true' if bucket.value is string. 
`
!Number.isInteger(Number('stringValue')); /// => true`

Reply:
I don't think this line would justify a utility method, especially as you mentioned it's not very generic, since it works because we know that this will be a valid number in this place, generated by TSVB.

Reply:
what about to use RegExp here instead?

Reply:
Why would you prefer using regex, if we can use a native Number method to check the same?

Reply:
I know that we "sure" that the number in this place is a valid number, but I never trust code, so I'd also like to have a small check for `NaN` values here. It doesn't hurt the code having one more check